x-pack/plugins/screenshotting/server/formats/index.ts

```diff
@@ -5,4 +5,6 @@
  * 2.0.
  */
 
-export { PdfMaker } from './pdfmaker';
+export * from './pdf';
+export * from './png';
```

Comment:
I think we need to avoid `export *`, unfortunately as it interferes with automated API documentation